Leanplum/Api/LPUserApi.m

```diff
@@ -49,11 +50,20 @@ + (void) setUserId:(NSString *)userId
     if (attributes != nil) {
         params[LP_PARAM_USER_ATTRIBUTES] =  [LPJSON stringFromJSON:attributes];
     }
-    LPWSManager *wsManager = [[LPWSManager alloc] init];
-    [wsManager sendPOSTWebService:[LPApiMethods getApiMethod:LPApiMethodSetUserAttributes]
-                       withParams:params
-                     successBlock:successResponse
-                     failureBlock:failureResponse];
+
+//    if ([LPApiConstants sharedState].isMulti) {
```

Comment:
Not sure this should be commented out. Also missing the logic for responseDictionaryFromResponse

Reply:
Missing the logic? What do you mean? It's there on L27, and the method is in `LPAPIUtils`